src/olympia/devhub/templates/devhub/verify_email.html

```diff
@@ -20,37 +21,68 @@ <h1>{{ title }}</h1>
   Could not verify email address. The verification link has expired.
   {% endtrans %}
 {% elif state == "verification_pending" %}
-  {% trans %}
-  Working... Please be patient.
-  {% endtrans %}
   <div class="loader"></div>
-{% elif state == "verification_failed" %}
-  {% trans %}
-  Failed to send confirmation email. Please try again.
-  If you no longer have access to your email address, please update your mozilla account email address.
-  <a
-    href="https://support.mozilla.org/en-US/kb/change-primary-email-address-firefox-accounts"
-    target="_blank"
-  >
-    Change email address
-  </a>
-  {% endtrans %}
+    {% trans %}
+    We are sending an email to you, this might take a minute. The page will automatically refresh.
+    {% endtrans %}
 {% elif state == "verification_timedout" %}
   {% trans %}
-  This is taking longer than expected. Try again.
+  It is taking longer than expected to confirm delivery of your verification email. Please try again.
   {% endtrans %}
 {% elif state == "confirmation_pending" %}
   {% trans email=request.user.email %}
   An email with a confirmation link has been sent to your email address: {{ email }}. Please click the link to confirm your email address. If you did not receive the email, please check your spam folder.
   {% endtrans %}
 {% elif state == "confirmation_invalid" %}
   {% trans email=request.user.email %}
-    The provided code is invalid. Please use the link in the email sent to your email address {{ email }}.
-    - could be unauthorized
-    - could be expired
-    - could be a mistake
+  The provided code is invalid, unauthorized, expired or incomplete. Please use the link in the email sent to your email address {{ email }}. If the code is still not working please request a new email.
```

Comment:
```suggestion
  The provided code is invalid, unauthorized, expired or incomplete. Please use the link in the email sent to your email address {{ email }}. If the code is still not working, please request a new email.
```

Probably worth a comma in the last sentence.